ichnaea/conftest.py

```diff
@@ -90,29 +90,30 @@ def package():
             print(obj)
 
 
-def _setup_table_contents(conn):
+def _setup_table_data(engine):
     # Avoid import cycle
     from ichnaea.tests.factories import ApiKeyFactory
 
-    conn.execute(text("DELETE FROM api_key"))
-    conn.execute(text("DELETE FROM export_config"))
-    key = ApiKeyFactory.build(valid_key="test")
-    state = dict(key.__dict__)
-    del state["_sa_instance_state"]
-    conn.execute(key.__table__.insert().values(state))
-
-
-def setup_tables(engine):
-    alembic_cfg = get_alembic_config()
     with engine.connect() as conn:
         with conn.begin() as trans:
-            # Now stamp the latest alembic version
-            command.stamp(alembic_cfg, "base")
-            command.upgrade(alembic_cfg, "head")
-            _setup_table_contents(conn)
+            conn.execute(text("DELETE FROM api_key"))
+            conn.execute(text("DELETE FROM export_config"))
+            key = ApiKeyFactory.build(valid_key="test")
+            state = dict(key.__dict__)
+            del state["_sa_instance_state"]
+            conn.execute(key.__table__.insert().values(state))
             trans.commit()
 
 
+def setup_tables(engine):
+    # Stamp the latest alembic version
+    alembic_main(["stamp", "base"])
+    alembic_main(["upgrade", "head"])
```

Comment:
Calling these like this will run alembic as if it was being run from the command line, so it picks up `alembic.ini` and `env.py` and all the bits.